asv/plugins/conda.py

```diff
@@ -87,6 +88,15 @@ def __init__(self, conf, python, requirements, tagged_env_vars):
         self._requirements = requirements
         self._conda_channels = conf.conda_channels
         self._conda_environment_file = conf.conda_environment_file
+
+        if conf.conda_environment_file == "IGNORE":
+            log.debug("Skipping environment file due to conda_environment_file set to IGNORE")
+            self._conda_environment_file = None
+        elif not conf.conda_environment_file:
+            if Path("environment.yml").exists():
+                log.debug("Using environment.yml")
+                self._mamba_environment_file = "environment.yml"
```

Comment:
Should this be `self._conda_environment_file`? It's the only use of `_mamba` in this module, so it looks like a C&P mistake.

Reply:
Yup, good catch.